app/src/main/res/values-it-rIT/strings.xml

```diff
@@ -338,6 +338,7 @@
     <string name="importer_help_steam">Seleziona una copia di <b>/data/data/com.valvesoftware.android.steam.community/files/Steamguard-*.json</b>, situata nella directory della memoria interna di Steam.</string>
     <string name="importer_help_totp_authenticator">Seleziona un file di esportazione di TOTP Authenticator.</string>
     <string name="importer_help_winauth">Seleziona un file di esportazione di WinAuth.</string>
+    <string name="importer_encrypted_exception_google_authenticator">Voce criptata:</string>
```

Comment:
This will get overwritten when we import changes from Crowdin. Could you remove it?